src/core/cistem_parameters.cpp

```diff
@@ -2096,7 +2096,7 @@ int cisTEMParameters::ReturnMaxPositionInStack(bool exclude_negative_film_number
 
 static int wxCMPFUNC_CONV SortByReference3DFilenameCompareFunction(cisTEMParameterLine** a, cisTEMParameterLine** b) // function for sorting the classum selections by parent_image_id - this makes cutting them out more efficient
 {
-    return wxStringSortAscending(&(*a)->reference_3d_filename, &(*b)->reference_3d_filename);
+    return wxStringSortAscending((*a)->reference_3d_filename, (*b)->reference_3d_filename);
```

Comment:
This change is required when building with latest wxwidgets, I don't know if you are pinning to a specific version in cisTEM. Please ignore if not relevant.